### DISCUSSION TITLE
umi不兼容还是不支持vue吗?

### DISCUSSION THREAD

**Discussion by singgl** [umijs/umi#11782] [General]
在中台无界界面嵌套umi子业务系统,发现路由参数routes会丢失

**Comment by fz6m** [comment]
umi 是一个 react 的面向企业中后台 pc 端的 SPA 框架，目前提供的 vue 、vite 都是实验性功能，只能独立小范围使用，无法开启其他 umi 功能，当你有规模性或复杂性的 vue 项目需求时，不建议使用 umi 。

**Reply by singgl** [reply]
也没有很复杂,就是在无界中台直接嵌套umi项目
